package.json

```diff
@@ -77,6 +77,7 @@
     "url": "https://github.com/rhinogram/rhinostyle"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.0.0",
```

Comment:
Let's take out the `^`

Reply:
I'm not sure if we want to mix babel deps in version 6 (existing babel deps) with the new version 7. Maybe this doesn't matter for polyfill? Just something to verify.